bazel/repositories_extra.bzl

```diff
@@ -97,6 +97,16 @@ def _python_deps():
         # release_date = "2020-05-21"
         # use_category = ["test"],
     )
+    pip3_import(
+        name = "fuzzing_pip3",
+        requirements = "@rules_fuzzing//fuzzing:requirements.txt",
+
+        # project_name = "Abseil Python Common Libraries",
+        # project_url = "https://github.com/abseil/abseil-py",
+        # version = "0.11.0",
+        # release_date = "2020-10-27",
+        # use_category = ["test"],
+    )
```

Comment:
This is a little different from our existing Python dependencies that normally have a `requirements.txt` file. The `requirements.txt` file allows us to pin to sha256 hashes with `extra_pip_args = ["--require-hashes"],`.

From what I can tell this is referring to https://github.com/bazelbuild/rules_fuzzing/blob/master/fuzzing/requirements.txt. Is there a way to pull in this dependency and pin to the sha256 hash (which is required by [policy](https://github.com/envoyproxy/envoy/blob/main/DEPENDENCY_POLICY.md#declaring-external-dependencies))?

Reply:
I've just created bazelbuild/rules_fuzzing#121 to address that.

Do you mean to keep using the `requirements.txt` file from the `@rules_fuzzing` dependency, or make a copy in this repository?

If it's the former, I will update the dependency version to incorporate this change and enable the `--require-hashes` flag here.

If it's the latter, I think a good location for this file would be `//test/fuzz:rules_fuzzing_requirements.txt`. WDYT?

Reply:
> I've just created [bazelbuild/rules_fuzzing#121](https://github.com/bazelbuild/rules_fuzzing/pull/121) to address that.
> 
> Do you mean to keep using the `requirements.txt` file from the `@rules_fuzzing` dependency, or make a copy in this repository?
> 
> If it's the former, I will update the dependency version to incorporate this change and enable the `--require-hashes` flag here.

This part is done - let me know if you'd prefer to go with the second approach instead.